include/crm/common/resources.h

```diff
@@ -24,6 +24,7 @@ extern "C" {
 enum rsc_start_requirement {
     pcmk_requires_nothing   = 0,    //!< Resource can be recovered immediately
     pcmk_requires_quorum    = 1,    //!< Resource can be recovered if quorate
+    pcmk_requires_fencing   = 2,    //!< Resource can be recovered after fencing
 
```

Comment:
Did neither check standards nor did I try but iirc it should be possible to combine explicit assignment with defaulting to increment by 1. But even if it is safe pros & cons ...

Reply:
And if you want to deprecate certain values but still wanna stay compatible later on ...
And most dangerous is probably the one case that sounds appealing at first: leave spaces and
use explicit assignment whenever you want to start a new sequence ... when you've eaten up
the free space there is probably nothing that would even warn you and I don't see a quick and
simple safeguarding measure ...